sys-devel/clang-common/clang-common-17.0.0.9999.ebuild

```diff
@@ -63,6 +63,30 @@ pkg_pretend() {
 	fi
 }
 
+doclang_cfg() {
+	local triple="${1}"
+	insinto /etc/clang
```

Comment:
Let's keep `insinto` in the main function, no point in calling it multiple times.

Reply:
Didn't realize it persisted between function calls. Will do.